contracts/cw-dao/src/contract.rs

```diff
@@ -1288,6 +1330,112 @@ mod tests {
         assert_eq!(ContractError::WrongCloseStatus {}, err.downcast().unwrap());
     }
 
+    #[test]
+    fn test_close_works_with_refund() {
+        let mut app = mock_app();
+        let voting_period = Duration::Height(2000000);
+        let threshold = Threshold::ThresholdQuorum {
+            threshold: Decimal::percent(51),
+            quorum: Decimal::percent(10),
+        };
+        let proposal_deposit_amount = Uint128::new(10);
+        let (dao_addr, _cw20_addr) = setup_test_case(
```

Comment:
If you're going to use this `_cw20_addr` variable like you do below, it shouldn't have the `_` in front of it. Should be `cw20_addr`.